rad-auth/src/main.rs

```diff
@@ -0,0 +1,53 @@
+use radicle_tools::cli;
+use rad_clib::keys::ssh::SshAuthSock;
+use cli::{keys, person, profile, tui};
+
+mod args;
+
+fn main() -> anyhow::Result<()> {
+    match run() {
+        Ok(()) => Ok(()),
+        Err(_) => {
+            println!();
+            std::process::exit(0);
```

Comment:
```suggestion
            std::process::exit(1);
```
Non-zero exit for errors.

Reply:
:facepalm: That sneaked in...